bin/cucumber-language-server.cjs

```diff
@@ -4,12 +4,15 @@ require('source-map-support').install()
 const { startStandaloneServer } = require('../dist/cjs/src/wasm/startStandaloneServer')
 const { NodeFiles } = require('../dist/cjs/src/node/NodeFiles')
 const url = require('url')
-const { version } = require('../src/version')
+const { version } = require('../dist/cjs/src/version')
 
 const wasmBaseUrl = url.pathToFileURL(
   `${__dirname}/../node_modules/@cucumber/language-service/dist`
 )
-const { connection } = startStandaloneServer(wasmBaseUrl.href, (rootUri) => new NodeFiles(rootUri))
+const { connection } = startStandaloneServer(
+  url.fileURLToPath(wasmBaseUrl),
+  (rootUri) => new NodeFiles(rootUri)
+)
```

Comment:
Can you provide more information on rationale to change from a `file:///` URL to a file path? I'm also wondering would there be impact if we wanted to run the language server in a browser-based context in future.

- Perhaps, is there any error output you can provide that indicates the reason for this change?

If we are no longer using the URL, and simply want to use a file path; then I would consider, can we use `path` instead? An improvement in any case would be to avoid calling functions inside the arguments of another function where possible, which can make it easier to debug based on line number of failures e.g. assign`url.fileURLToPath(wasmBaseUrl)` to a variable, and then reference it from function call 👍 Below is just a quick reference

```javascript
const path = require('path');
const distPath = path.resolve(`${__dirname}/../node_modules/@cucumber/language-service/dist`)
const { connection } = startStandaloneServer(distPath, (rootUri) => new NodeFiles(rootUri))
```

Has this been tested on Windows? Has it been tested through VSCode - see [contributing guidelines](https://github.com/cucumber/vscode/blob/main/CONTRIBUTING.md)? Would like to ensure works with Windows paths. Are there any unit tests that could be added for this module that could provide us with more confidence in changes to this file now and going forward?

Reply:
Hi @kieran-ryan,

Sorry about the delay in my response. I don't use Windows regularly, so it took me some time to set up an environment for testing. I've now tested the PR on Windows, but only with Node.js v16.x.x. This is due to compatibility issues with the latest release of the `language-client` package on newer Node.js versions.

I've also updated the PR description to provide more context on the reasons behind these changes. Please refer to the updated description for a clearer understanding of their value.

Thank you for your patience and understanding.

Reply:
Hi @kieran-ryan,

I'd suggest omitting setup documentation for Neovim users within this PR, as it's already comprehensively covered in the Neovim community documentation at [https://github.com/neovim/nvim-lspconfig/blob/master/doc/server_configurations.md#cucumber_language_server](https://github.com/neovim/nvim-lspconfig/blob/master/doc/server_configurations.md#cucumber_language_server).

Regarding unit testing, effectively testing this functionality requires a real LSP client. I'll address this in a separate PR, if deemed necessary, to ensure thorough testing.